projects/components/src/load-async/wrapper/load-async-wrapper.component.ts

```diff
@@ -39,9 +40,13 @@ export class LoadAsyncWrapperComponent {
 
   public content?: TemplateRef<LoadAsyncContext>;
 
+  private readonly loaderTypeSubject: BehaviorSubject<LoaderType> = new BehaviorSubject<LoaderType>(LoaderType.Spinner);
```

Comment:
We are setting different default options at multiple places. It would be useful if we set it at only one place

Reply:
then, I need to put undefined as the initial value.